editor/inspector_dock.cpp

```diff
@@ -446,8 +461,19 @@ void InspectorDock::update(Object *p_object) {
 	p->clear();
 	p->add_icon_shortcut(get_icon("GuiTreeArrowDown", "EditorIcons"), ED_SHORTCUT("property_editor/expand_all", TTR("Expand All")), EXPAND_ALL);
 	p->add_icon_shortcut(get_icon("GuiTreeArrowRight", "EditorIcons"), ED_SHORTCUT("property_editor/collapse_all", TTR("Collapse All")), COLLAPSE_ALL);
-	p->add_separator();
 
+	p->add_separator(TTR("Property Name Style"));
+	p->add_radio_check_item(TTR("Raw"), PROPERTY_NAME_STYLE_RAW);
+	p->add_radio_check_item(TTR("Capitalized"), PROPERTY_NAME_STYLE_CAPITALIZED);
+	p->add_radio_check_item(TTR("Localized"), PROPERTY_NAME_STYLE_LOCALIZED);
+
+	if (!EditorPropertyNameProcessor::is_localization_available()) {
+		const int index = p->get_item_index(PROPERTY_NAME_STYLE_LOCALIZED);
+		p->set_item_disabled(index, true);
+		p->set_item_tooltip(index, TTR("Localization not available for current language."));
+	}
```

Comment:
That doesn't seem to work for me on Linux with the `en` locale, the option is still enabled.

Reply:
https://github.com/godotengine/godot/blob/8ecdb0b6e93e165b030e791366db551c8a5d531f/editor/editor_property_name_processor.cpp#L54-L57

This is based on `interface/editor/editor_language`. I tried launching the editor using `LANG=en_US`, the editor setting is "en", and the option is disabled. What's your `interface/editor/editor_language`?

Reply:
Hm I just tested again and it's working fine now, weird.